packages/messenger-widget/src/components/MessageInputBox/bl.ts

```diff
@@ -151,6 +186,42 @@ const editMessage = async (
     const haltDelivery = getHaltDelivery(state);
     const sendDependencies: SendDependencies = getDependencies(state, account);
 
+    try {
+        const { encryptedEnvelop } = await buildEnvelop(
```

Comment:
why do we have the same function in 105 and here?

Reply:
Different actions of messages like , new , edit, reply has separate method and all of them checks the message size limit before sending so..

Reply:
Than please move the check into a separate function to avoid code duplication